src/App.tsx

```diff
@@ -15,36 +16,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  alphabet = 'alphabet',
+  length = 'length',
+}
+
+function sortGoods(
+  goods: string[],
+  sortField: SortField | '',
+  reverse: boolean,
+) {
+  const changedGoods = [...goods];
+
+  if (sortField) {
```

Comment:
The `if (sortField)` condition checks if `sortField` is truthy, which works fine for the current implementation. However, since `sortField` can be an empty string, it might be more explicit to check `if (sortField !== '')` to ensure clarity and avoid unnecessary sorting operations when `sortField` is an empty string.